src/main/java/com/questhelper/runeliteobjects/extendedruneliteobjects/RuneliteObjectManager.java

```diff
@@ -894,7 +956,7 @@ else if (event.getGameState() == GameState.LOADING)
 			runeliteObjectGroups.forEach((groupID, extendedRuneliteObjectGroup) -> {
 				for (ExtendedRuneliteObject extendedRuneliteObject : extendedRuneliteObjectGroup.extendedRuneliteObjects)
 				{
-					disableObject(extendedRuneliteObject);
+					extendedRuneliteObject.render();
```

Comment:
This feels like a typo to me - previously this would call `disable()` on all objects

Reply:
The checks per object have been shifted into the object's person render checks for this, might be worth me changing the naming of render to imply that it checks for hide and show.